File: Coding Ninjas - CodeStudio/Basics Of Javascript Guided Path/factorial_of_a_number.js
function fact(n){
	if(n < 0) {
        return 'Error';
    }
    if(n == 0) {
        return 1;
    }
    let f = 1;
    for(let i = 2; i <= n; i++) {
        f *= i;
    }
    return f;
}

File: Coding Ninjas - CodeStudio/Basics of Java Guided Path/factorial_of_a_number.java
import java.util.Scanner;
class Solution {
	
	public static void main(String args[]) {
		
		Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();
        
        if(n < 0) {
            System.out.println("Error");
            return;
        }

        int fact = 1;
        
        while(n > 1) {
            fact *= n;
            n--;
        }
        
        System.out.println(fact);
		
	}
}